components/Badge/badge.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
You can add some styling to the badges as well. Try to make them cover a part of the image. Add different background to highlight a badge.